src/cascadia/TerminalApp/VsSetupConfiguration.h

```diff
@@ -0,0 +1,142 @@
+/*++
+Copyright (c) Microsoft Corporation
+Licensed under the MIT license.
+
+Module Name:
+- VsSetupConfiguration
+
+Abstract:
+- Encapsulates the Visual Studio Setup Configuration COM APIs
+
+Author(s):
+- Charles Willis - October 2020
+
+--*/
+
+#pragma once
+
+#include <wil/resource.h>

```

Comment:
I think this one's already in `pch.h` (by way of `LibraryIncludes.h`), so I don't think you need it here